egpu-switcher

```diff
@@ -287,6 +287,32 @@ switch() {
 		fi
 	fi
 
+    if [ $1 = "egpu" ] && [ $(grep -ce "nvidia" $XFILE_EGPU) -eq 0 ]; then
+        EGPU_PCI_ID=$(cat $XFILE_EGPU | grep -Ei "BusID" | grep -oEi '[0-9]+\:[0-9]+\:[0-9]+')
+		# create an array by splitting the BUS-ID on ':'
+		busArray=(${EGPU_PCI_ID//:/ })
+		bus1d=${busArray[0]}
+		bus2d=${busArray[1]}
+		bus3d=${busArray[2]}
+		# convert dec to hex
+		bus1h=$(printf "%02x" $bus1d)
+		bus2h=$(printf "%02x" $bus2d)
+		bus3h=$(printf "%01x" $bus3d)
+		HEX_ID=$(echo $bus1h:$bus2h.$bus3h)
+    	DISPLAY_ON="false"
+    	for DISP in $(cat /sys/bus/pci/devices/0000:${HEX_ID}/drm/card[0-9]*/card[0-9]*-*/status); do
+      		if [ ${DISP} = "connected" ]; then
+        		DISPLAY_ON="true"
+      		fi
+    	done
+    	if [ ${DISPLAY_ON} = "false" ]; then
+      		echo "Warning: No eGPU attached display detected with open source drivers"
+      		echo "Internal mode and setting DRI_PRIME variable are recommended for this configuration"
+      		echo "Not setting eGPU mode."
```

Comment:
Maybe change this to the following code:

```bash
echo -e "$WARN No eGPU attached display detected with open source drivers. Internal mode and setting DRI_PRIME variable are recommended for this configuration."
echo -e "$WARN -> Not setting eGPU mode."
```

Just to be consistent with the output formatting :)